### PR TITLE
JavaScript - Synchronous Event Delivery

### PR DESCRIPTION
**Pull request by kapnobatai137**
- update output of code

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/node/events/synchronous-event-delivery.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/node/events/synchronous-event-delivery.md?ref=cc23f67b0adf776cf2343f82b2c3cf7aa9ee25c0)

File: javascript/node/events/synchronous-event-delivery.md [hunk 1]
```diff
@@ -1,60 +1,61 @@
...
-emitter.emit(‘hi’);
-console.log(‘post hi’);
+console.log('pre hi');
+emitter.emit('hi');
+console.log('post hi');
 ```
 
 Gives the following output:
 
 ```bash
```
Comment (kapnobatai137) [inline javascript/node/events/synchronous-event-delivery.md:50]:
```suggestion
```plain-text
```

File: javascript/node/events/synchronous-event-delivery.md [hunk 2]
```diff
@@ -1,60 +1,61 @@
...
 links:
   - >-
     [Synchronous Event
     Delivery](https://blog.yld.io/2015/12/15/using-an-event-emitter/#.WI4ybbaLQy4){website}
+
 parent: listening-to-events-just-once
+
 ---
 
 # Event delivery
```
Comment (kapnobatai137) [inline javascript/node/events/synchronous-event-delivery.md:21]:
```suggestion
# Event Delivery
```

**Pull request merged**